src/main/resources/bidder-config/teqblaze.yaml

```diff
@@ -19,6 +19,25 @@ adapters:
             url: https://usync.rocketlab.ai/pbserver?gdpr={{gdpr}}&consent={{gdpr_consent}}&us_privacy={{us_privacy}}&gpp={{gpp}}&gpp_sid={{gpp_sid}}&redirect={{redirect_url}}
             support-cors: false
             uid-macro: '[UID]'
+      appStockSSP:
+        enabled: false
+        # We have the following regional endpoint domains:  'lb' - for US_EAST, 'ortb-eu' - for EU, 'ortb-apac' - for APAC
+        # Please deploy this config in each of your datacenters with the appropriate regional subdomain
+        # Endpoint pattern: https://{REGION}.al-ad.com/pserver
+        endpoint: https://lb.al-ad.com/pserver
```

Comment:
I prefer to do it like in Go rather than specifying the complete URL for certain region